From 4edfee6bd2be248de18d0c865be87cb0f72ddd28 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 16 Aug 2019 06:30:35 -0400
Subject: [PATCH] limit product name less then 30 chars

Previously 40 is still too long, which broke gui window layout

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/storage/root.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/storage/root.py b/pyanaconda/storage/root.py
index a321899..4e3b7aa 100644
--- a/pyanaconda/storage/root.py
+++ b/pyanaconda/storage/root.py
@@ -160,7 +160,7 @@ def get_release_string():
         if os.access(filename, os.R_OK):
             (rel_name, rel_ver) = _release_from_os_release(filename)
 
-    return rel_arch, rel_name, rel_ver
+    return rel_arch, rel_name[0:30], rel_ver
 
 
 def _release_from_redhat_release(fn):
-- 
2.8.1

